src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -609,33 +654,11 @@ public virtual async Task PatchOneUpdateInAccessibleRowWithDatabasePolicy()
                     operationType: Config.Operation.UpsertIncremental,
                     requestBody: requestBody,
                     exceptionExpected: true,
-                    expectedErrorMessage: $"Cannot perform INSERT and could not find {_foreignKeyEntityName} with primary key <id: 1234> to perform UPDATE on.",
-                    expectedStatusCode: HttpStatusCode.NotFound,
-                    expectedSubStatusCode: DataApiBuilderException.SubStatusCodes.EntityNotFound.ToString(),
+                    expectedErrorMessage: $"Authorization Failure: Access Not Allowed.",
```

Comment:
>Authorization Failure: Access Not Allowed.", [](http://example.com/codeflow?start=44&length=44)

can you update the method summary to note whether this authorization failure is for update or insert